<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>450</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>400</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Панков Василий Дмитриевич, группа М412</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="mainLayout">
    <property name="spacing">
     <number>15</number>
    </property>
    <property name="leftMargin">
     <number>20</number>
    </property>
    <property name="topMargin">
     <number>20</number>
    </property>
    <property name="rightMargin">
     <number>20</number>
    </property>
    <property name="bottomMargin">
     <number>20</number>
    </property>
    <item>
     <widget class="QLabel" name="titleLabel">
      <property name="font">
       <font>
        <pointsize>14</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Расчет стоимости компьютера</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="baseGroup">
      <property name="title">
       <string>Основное изделие</string>
      </property>
      <layout class="QVBoxLayout" name="baseLayout">
       <item>
        <widget class="QLabel" name="baseLabel">
         <property name="text">
          <string>Компьютер: 50000 руб (всегда включен)</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="componentsGroup">
      <property name="title">
       <string>Комплектующие</string>
      </property>
      <layout class="QVBoxLayout" name="componentsLayout">
       <item>
        <widget class="QCheckBox" name="monitorCheckBox">
         <property name="text">
          <string>Монитор: 15000 руб</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="keyboardCheckBox">
         <property name="text">
          <string>Клавиатура: 3000 руб</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="mouseCheckBox">
         <property name="text">
          <string>Мышь: 2000 руб</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="speakersCheckBox">
         <property name="text">
          <string>Колонки: 5000 руб</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="resultGroup">
      <property name="title">
       <string>Результат расчета</string>
      </property>
      <layout class="QVBoxLayout" name="resultLayout">
       <item>
        <widget class="QLabel" name="resultLabel">
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="text">
          <string>Результаты расчета появятся здесь</string>
         </property>
         <property name="wordWrap">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <spacer name="verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
